content/en/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,390 @@
+---
+reviewers:
+- sftim
+title: LimitRange
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on kubernetes cluster.
+With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
+Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.
+there is a concern that one Pod/Container could monopolise its namespace resources.
+LimitRange is a tool to address this concern.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+A limit range, defined by a `LimitRange` object, provides constraints that can :
+- Enforce min/max compute resources usage per Pod/Container in a namespace
+- Enforce min/max storage request per PersistentVolumeClaim in a namespace
+- Enforce a ratio between request and limit for a resource in a namespace
+- Set default request/limit for compute resources in a namespace and automatically inject them to Containers at runtime 
+
+
+Limit Range work like this:
+
+- The administrator creates one `LimitRange` in one namespace.
+- Users create resources (Pods, Containers, PersistentVolumeClaim) in the namespace,
+  the `LimitRanger` admission controller force defaults requests/limits if any for Pods/Container that make no compute resource requirements,
+  tracks usage to ensure it does not exceed  resource min/max defined in any  `LimitRange` present in the namespace.

```

Comment:
line 32, clean up the wording in list item that starts with `Users`, I am unable to parse it.
the `LimitRanger` admission controller ... forces default requests

You may want to link to `Enabling Limit Range` below, as this is the first time the admission controller is mentioned.

Reply:
yes, it make sens. Reorganized whole section section.